projects/models.py

```diff
@@ -8,7 +8,22 @@ class ModelBase(models.Model):
     class Meta:
         abstract = True
 
+class Client(models.Model):
+    department = models.CharField(max_length=255, blank=True)
+    agency = models.CharField(max_length=255, blank=True)
+    omb_agency_code = models.CharField(max_length=255, blank=True)
+    omb_bureau_code = models.CharField(max_length=255, blank=True)
+    treasury_agency_code = models.CharField(max_length=255, blank=True)
+    cgac_agency_code = models.CharField(max_length=255, blank=True)
+
+    def __str__(self):
+        return '%s - %s' % (self.department, self.agency)
+
+    class Meta:
+        verbose_name = 'Client'
```

Comment:
ah! wanted to make sure that the `Client` class was displayed correctly in the admin panel, specifically on the front page of the panel it was specified as `Clients`. i dropped out the `Meta` sub-class and it still looks fine, so looks like it was unnecessary :-).